test/integration/compose/inspec/compose_spec.rb

```diff
@@ -1,4 +1,4 @@
 describe command('/usr/local/bin/docker-compose version') do
   its('exit_status') { should eq 0 }
-  its('stdout') { should match(/docker-compose version 1\.18\.0/) }
+  its('stdout') { should match(/docker-compose version 2\.15\.0/) }
```

Comment:
You need to fix the test to do the following.
```suggestion
  its('stdout') { should match(/docker-compose version v2\.15\.0/) }
```